process/banning.html

```diff
@@ -44,37 +44,55 @@ <h4>Nearby</h4>
 <ul>
 <li>from a single group, to the CEO;</li>
 <li>from a single group that the individual has not formally joined, to the chair(s) of the group; and</li>
 <li>from all W3C groups, to the CEO and COO jointly.</li>
 </ul>
 <p>If the chair, team contact, CEO, COO is the one accused of unacceptable behavior, they should not be involved in the decision to ban.</p>
-
-<p>2. Following violation of the <a href="https://www.w3.org/Consortium/cepc/">W3C Code of Ethics and Professional Conduct</a>, at least one warning by the chair(s), and at least one subsequent violation of the CEPC, chairs may temporarily or permanently suspend participation by a person who has not formally joined the group.</p>
-
-<p>3. A W3C group chair and team contact may propose to suspend or remove a participant from a single group - typically for egregious and repetitive CEPC violations - by making a request to the CEO.</p>
+</li>
+<li><p>In case of disagreement between the W3C group chair and team contact, each should explain their reasons to the CEO.</p></li>
+<li>
+<p>Following violation of the <a href="https://www.w3.org/Consortium/cepc/">W3C Code
+of Ethics and Professional Conduct</a>, a W3C group chair and team contact should
+give at least one written warning. If the person is a is a W3C Member representative
+participant in the Group, the Member's Advisory Committee representative must be
+notified.</p>
+</li>
+<li>
+  <p>Upon subsequent violation of the CEPC, a W3C group chair and team contact may:</p>
+  <ol>
+<li>
+  <p>temporarily or permanently suspend participation by a person if the person has not formally joined the group.</p>
+</li>
+<li>
+<p>propose to suspend or remove a participant from a single group - typically for egregious and repetitive CEPC violations - by making a request to the CEO.</p>
       <ul>
-        <li>If the group chair and team contact are not in agreement, each should explain their reasons.</li>
         <li>If the participant is a W3C Member representative, the CEO will coordinate the response with the Member's Advisory Committee representative.</li>
         <li>If anyone raises an issue to the CEO that a participant fails to meet the requirements of <a href="https://www.w3.org/Consortium/Process/#ParticipationCriteria">Individual Participation Criteria</a>, then the CEO is empowered to investigate. If the participant indeed is failing to meet those requirements, then after at least one warning and at least one subsequent violation, the CEO may temporarily or permanently suspend participation.</li>
+        <li>To avoid potential rapid escalation of unacceptable behavior, a W3C group chair and team contact may temporarily suspend a participant from a single group for up to 5 business days after alerting the CEO (and the Member's Advisory Committee representative if the participant is not an Invited Expert). Any subsequent suspension within 6 months would need to be requested from the CEO.</li>
```

Comment:
Consider emphasizing that the chair and team contact must act together: "... a W3C group chair together with a team contact may ..."